sources/packages/web/src/components/common/OfferingView.vue

```diff
@@ -0,0 +1,156 @@
+      {{ offeringViewData.offeringName }}
+    </h3>
+    <v-row>
+      <v-col cols="12">
+        <title-value
+          propertyTitle="Location"
+          :propertyValue="offeringViewData.locationName"
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Study intensity"
+          :propertyValue="offeringViewData.offeringIntensity"
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Study delivery"
+          :propertyValue="offeringViewData.offeringDelivered"
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Study dates"
+          :propertyValue="
+            dateOnlyLongPeriodString(
+              offeringViewData.studyStartDate,
+              offeringViewData.studyEndDate,
+            )
+          "
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value propertyTitle="Study breaks">
+          <template #value>
+            <ul class="no-bullets">
+              <li
+                v-for="studyBreak in offeringViewData.studyBreaks"
+                :key="studyBreak.breakStartDate"
+              >
+                {{
+                  dateOnlyLongPeriodString(
+                    studyBreak.breakStartDate,
+                    studyBreak.breakEndDate,
+                  )
+                }}
+              </li>
+            </ul>
+          </template>
+        </title-value>
+      </v-col>
+    </v-row>
+    <h3 class="category-header-medium primary-color mt-6">Study costs</h3>
+    <v-row>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Tuition"
+          :propertyValue="formatCurrency(offeringViewData.actualTuitionCosts)"
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Program related costs"
+          :propertyValue="formatCurrency(offeringViewData.programRelatedCosts)"
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Mandatory fees"
+          :propertyValue="formatCurrency(offeringViewData.mandatoryFees)"
+        />
+      </v-col>
+      <v-col cols="12" sm="6">
+        <title-value
+          propertyTitle="Exceptional expenses"
+          :propertyValue="formatCurrency(offeringViewData.exceptionalExpenses)"
+        />
+      </v-col>
+    </v-row>
+  </content-group-info>
+</template>
+
+<script lang="ts">
+import { useFormatters } from "@/composables";
+import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
+import { EducationProgramOfferingSummaryViewAPIOutDTO } from "@/services/http/dto";
+import { defineComponent, watch, ref } from "vue";
+
+export default defineComponent({
+  props: {
+    offeringId: {
+      type: Number,
+      required: false,
+    },
+    locationId: {
+      type: Number,
+      required: false,
```

Comment:
shouldn't these 2 props be required?

Reply:
I think Location may not be(if it is for ministry) but offeringId is required.

Reply:
Location id will be required only for institutions.
The offering id is not available till an API call returns it.